### DISCUSSION TITLE
Actividad 0 - Milocco, Valentin

### DISCUSSION THREAD

**Discussion by ValentinMilocco** [frlp-utn-ingsoft/catedra#79] [Show and tell]
https://github.com/ValentinMilocco/MiRepositorio.git

**Comment by RodrigoJacznik** [comment]
Buenas,
Los commits podrían ser mas atómicos.
Falta la parte 4

**Reply by ValentinMilocco** [reply]
Buenas, ayer hice los commits pertinentes a la parte 4.
saludos

**Reply by ivanduva** [reply]
Perfecto Valentín, queda aprobado.
Saludos!
